pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -136,12 +140,18 @@ func getIgnitionConfig(assetsDir string, profile *performancev2.PerformanceProfi
 
 	// add crio config snippet under the node /etc/crio/crio.conf.d/ directory
 	crioConfdRuntimesMode := 0644
+	templateArgs := make(map[string]string)
```

Comment:
Better, but I'd really do more GetOCIHooks... and move this inside addCrioConfigSnippet

Reply:
done